rocky/reports/views/base.py

```diff
@@ -130,22 +130,22 @@ def get_report_types_for_aggregate_report(
     def get_required_optional_plugins(
         self, plugin_ids_dict: dict[str, set[str]]
     ) -> tuple[dict[str, list[Plugin]], dict[str, bool]]:
-        plugins = {}
-        for plugin, plugin_ids in plugin_ids_dict.items():
-            plugins[plugin] = [get_katalogus(self.organization.code).get_plugin(plugin_id) for plugin_id in plugin_ids]
+        all_plugins = {}
+        all_plugins_enabled = {}
```

Comment:
I think within this context the names `plugins` and `enabled_plugins` would make more sense

Reply:
There a list also called plugins, but renamed to `enabled_plugins `